### ISSUE TITLE
Character encoding problem when annotating

### ISSUE THREAD

**Issue by freg1086** [stormliucong/doc2hpo#29]
Hi,

First of all, thank you for this wonderful tool.

I am a clinical genetisist, and I use doc2hpo to manually annotate our clinical reports. 
The dictionary look-up feature is very nice and allows me to save up a lot of time!

I have deployed a local doc2hpo server on my laptop (macOS X, 10.11.6). All seems to work nice, but, unfortunately, some characters in my reports are dissappearing in the parsing results when I submit the text. 
The characters that seem to cause problems are specific such as é à è etc. Example included. 
![doc2hpo](https://user-images.githubusercontent.com/36641354/99957866-a57a7400-2d88-11eb-8411-d8cbd58621bb.png)

If I understand correctly the problem comes from the fact that the default encoding for doc2hpo is UTF-8 ( which can have some trouble with é à è characters (not sure about that however).

Any idea how can I resolve the problem ?

Thanks a lot for your help.

**Comment by stormliucong** [comment]
Yes, the default encoding for doc2hpo is UTF-8. We currently support English annotation. It is a great idea to expand to other languages. We will investigate it and see how we can solve this issue. Thank you for reporting this.
